src/lib/types.ts

```diff
@@ -34,3 +36,11 @@ export interface ChannelData {
     name: string;
     private: boolean;
 }
+
+export interface CommentData {
+    user: uuid; // may need to be User
+    content: string;
+    downvotes: number;
+    upvotes: number;
+    children?: CommentData[];
```

Comment:
I would prefer to have the complete `User` type from the database stored here as it contains information that we'll use to display like the name and profileImage.

Would also like the `Comment` type here as well, since that contains all metadata about a comment that we may want to show.